lib/hard_data.dart

```diff
@@ -1,17 +1,15 @@
-// A list of two cards
-import 'package:flutter/material.dart';
+// import 'package:flutter/material.dart';
```

Comment:
nit: prefer deleting outright to commenting out. We use source control for a reason =)

Reply:
Doh! thanks for spotting that @domesticmouse , yes that's a plain mistake I'm the same - no leaving commented out code!
Unfort this has been merged, I'll make sure to fix that in my next PR !